docs/reference/run.md

```diff
@@ -629,6 +631,12 @@ command:
 
     $ docker run --security-opt label:type:svirt_apache_t -it centos bash
 
+If you want to prevent your container processes from gaining additional
+privileges, you can execute the following command:
```

Comment:
Added a link to kernel documenation which is the best source for more information.